src/Data/Uniques/Special/New.lua

```diff
@@ -10,18 +10,93 @@ Faithguard
 Runic Helm
 League: Expedition
 Requires Level 25, 30 Str, 30 Dex, 30 Int
-+30 to Intelligence
-25% increased Ward
-26% faster Restoration of Ward
-23% increased light radius
++(20-30) to Intelligence
+(25-35)% increased Ward
+(20-30)% faster Restoration of Ward
+(15-25)% increased light radius
 Increases and Reductions to maximum Energy Shield instead apply to Ward
 ]],[[
 Cadigan's Crown
 Runic Crown
 League: Expedition
 Requires Level 68, 66 Str, 66 Dex, 66 Int
 Never deal Critical Strikes
+Nearby Enemies cannot deal Critical Strikes
 Battlemage
+]],[[
+Elixir of the Unbroken Circle
```

Comment:
This flask seems to also give 200 flat Ward, although I'm not sure if that's while its active or while it's equipped, since this mod is placed at the top. Same with the other new unique flasks.

Reply:
That is part of their Iron Flask base type, and only when active.